### ISSUE TITLE
Does anyone use dropout rates other than 0.1 for training?

### ISSUE THREAD

**Issue by youmebangbang** [NVIDIA/tacotron2#471]
I've noticed reducing dropouts to around .07-.08 towards the end of training will lower training loss while slightly increasing validation loss. Not sure if model quality is better though.
